zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/RequestMapperTest.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway.rest;
+
+import org.junit.jupiter.api.Test;
+
+public class RequestMapperTest {
+
+  @Test
+  void shouldMapTenantToRequest() {}
```

Comment:
❓ Is this intended to be empty?

Reply:
Nope, left over, I initially wanted to add unit tests because I thought the logic was more complex, but :shrug:

Reply:
We have a follow up anyway to improve test coverage, bzw. unit tests here and in the client